app/web/features/dashboard/Hero/HeroSearch.tsx

```diff
@@ -0,0 +1,60 @@
+import { Typography } from "@material-ui/core";
+import LocationAutocomplete from "components/LocationAutocomplete";
+import { DASHBOARD } from "i18n/namespaces";
+import { useRouter } from "next/router";
+import { useTranslation } from "next-i18next";
+import { useForm } from "react-hook-form";
+import { routeToSearch } from "routes";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  searchBoxContainer: {
+    padding: theme.spacing(4, 2, 6, 2),
+    borderRadius: theme.shape.borderRadius,
+    backgroundColor: theme.palette.background.paper,
+  },
+}));
+
+export default function HeroSearch() {
+  const { t } = useTranslation(DASHBOARD);
+  const classes = useStyles();
+  const router = useRouter();
+  const searchInputId = "hero-search-input"; // @todo: replace with React 18's useId
```

Comment:
not really, but I don't think it should be hardcoded to a fixed value either. I'll remove the `todo` comment though as using `useId` isn't really necessary here because we don't expect to render this component more than once in a page. If that ever changes, the dev can make the change (once we're in v18)